ssc/cmod_pvsamv1.cpp

```diff
@@ -1165,6 +1181,26 @@ void cm_pvsamv1::exec()
         p_load_full = p_load_in;
     }
 
+    if (en_batt) {
+        if (p_load_forecast_in.size() > 0) {
```

Comment:
same here for size of p_load_forecast_in

Reply:
See https://github.com/NREL/ssc/blob/fcff96bd16bf1a8dbc38966a94bca69d2ab99ec0/ssc/cmod_pvsamv1.cpp#L1084 for the length constraints that have been enforced prior to this line.